packages/core/core-flows/src/order/workflows/receive-return.ts

```diff
@@ -0,0 +1,53 @@
+import { OrderWorkflow } from "@medusajs/types"
+import {
+  WorkflowData,
+  createStep,
+  createWorkflow,
+} from "@medusajs/workflows-sdk"
+import { useRemoteQueryStep } from "../../common"
+
+import { ReturnDTO } from "@medusajs/types/src"
```

Comment:
todo: can you check that we dont have other imports from src directly please 😄

Reply:
nice catch...

Reply:
I replaced from the entire project. there were many.

Reply:
nice catch 💪 I was almost sure some must have gone through, maybe worth checking dist as well 😓